parser/raylib_parser.c

```diff
@@ -610,17 +610,47 @@ int main(int argc, char* argv[])
     // funcs[]   -> We have all the functions decomposed into pieces for further analysis
     // defines[] -> We have all the defines decomposed into pieces for further analysis
 
-    // Process input file to output
-    if (outFileName[0] == '\0') MemoryCopy(outFileName, "raylib_api.txt\0", 15);
+
 
     printf("\nInput file:       %s", inFileName);
-    printf("\nOutput file:      %s", outFileName);
-    if (outputFormat == DEFAULT) printf("\nOutput format:    DEFAULT\n\n");
-    else if (outputFormat == JSON) printf("\nOutput format:    JSON\n\n");
-    else if (outputFormat == XML) printf("\nOutput format:    XML\n\n");
-    else if (outputFormat == LUA) printf("\nOutput format:    LUA\n\n");
 
-    ExportParsedData(outFileName, outputFormat);
+    if (outputFormat == ALL)
+    {
+        // Process input file to output
+        if (outFileName[0] == '\0') MemoryCopy(outFileName, "raylib_api\0", 11);
+
+        char outFileNameAll[512];
+        sprintf_s(outFileNameAll, 512, "txt/%s%s", outFileName, ".txt");
```

Comment:
I'm afraid raylib-parser avoids usage of string functions.

Reply:
Accessing the individual chars and changing them as required.

Reply:
Also note that `sprintf_s()` is a Microsoft specific function, not available on most C compilers.